ng2-material/components/data_table/data_table.ts

```diff
@@ -1,5 +1,5 @@
 import {Component, Input, Output, EventEmitter, ContentChildren, QueryList} from "angular2/core";
-import {MdDataTableTr} from "./data_table_tr";
+import {MdDataTableHeaderRow, MdDataTableRow, ITableRow} from "./data_table_tr";
 import "rxjs/add/operator/share";
 
```

Comment:
I would have thought this directive import / directives block would allow you not to add these rows to the `MATERIAL_DIRECTIVES` ... I guess because you are actually using them in the template of the example component you have to import them there?